storage/feed.go

```diff
@@ -79,6 +79,34 @@ type StartStoreEvent struct {
 	StoreID proto.StoreID
 }
 
+// StoreStatusEvent contains the current descriptor for the given store.
+//
+// Because the descriptor contains information that cannot currently be computed
+// from other events, this event should be periodically broadcast by the store
+// independently of other operations.
+type StoreStatusEvent struct {
+	StoreID proto.StoreID
+	Desc    *proto.StoreDescriptor
```

Comment:
It still wouldn't help, the status module doesn't see a `StoreEventFeed`, it just has a subscription to a generic feed.  `StoreEventFeed` is just a helper class for constructing and publishing store-specific events.